docs/getting-started-guides/clc.md

```diff
@@ -0,0 +1,317 @@
+# Kubernetes on CenturyLink Cloud
+These scripts handle the creation, deletion and expansion of kubernetes clusters on CenturyLink Cloud.
+
+You can accomplish all these tasks with a simple single command. And, for those interested in what's under the covers, we used Ansible to perform these tasks and we have made these Ansible playbooks available as well.
+
+## Find Help
+If you run into any problems or want help with anything, we are here to help. Reach out to use via any of the following ways:
+- Submit a github issue
+- Send an email to kubernetes AT ctl DOT io
+- Visit http://info.ctl.io/kubernetes
+
+## Clusters of VMs or Physical Servers, your choice.
+- We support Kubernetes clusters on both Virtual Machines or Physical Servers. If you want to use physical servers for the worker nodes (minions), simple use the --minion_type=bareMetal flag.
+- For more information on physical servers, visit: [https://www.ctl.io/bare-metal/](https://www.ctl.io/bare-metal/))
+- Physical serves are only available in the VA1 and GB3 data centers.
+- VMs are available in all 13 of our public cloud locations
+
+## Requirements
+The requirements to run this script are:
+- A linux administrative host (tested on ubuntu and OSX)
+- python 2 (tested on 2.7.11)
+  - pip (installed with python as of 2.7.9)
+- git
+- A CenturyLink Cloud account with rights to create new hosts
+- An active VPN connection to the CenturyLink Cloud from your linux host
+
+## Script Installation
+After you have all the requirements met, please follow these instructions to install this script.
+
+1) Clone this repository and cd into it.
+
+```
+git clone https://github.com/CenturyLinkCloud/adm-kubernetes-on-clc
+```
+
+2) Install all requirements, including
+  * Ansible
+  * CenturyLink Cloud SDK
+  * Ansible Modules
+
+```
+sudo pip install -r ansible/requirements.txt
+```
+
+3) Create the credentials file from the template and use it to set your ENV variables
+
+```
+cp ansible/credentials.sh.template ansible/credentials.sh
+vi ansible/credentials.sh
+source ansible/credentials.sh
+
+```
+
+4) Make sure the computer you are working on has access to the CenturyLink Cloud
+network. This is done by using a VM inside the CenturyLink Cloud network or
```

Comment:
Shorten to: 

Grant your machine access to the CenturyLink Cloud network by using a VM inside...